### DISCUSSION TITLE
Wrong prowler version in docker image?

### DISCUSSION THREAD

**Discussion by dsech** [prowler-cloud/prowler#1130] [General]
Hello, I've just discovered prowler, so I'm not sure if this is the correct place to report this.

The version of prowler doesn't match the docker image tag:
```
$ docker run --rm toniblyx/prowler:2.9.0 -V
Prowler 2.8.1-25March2022
```
I'm happy to open a bug report if needed, thanks!

**Comment by jfagoagas** [comment]
> Hello, I've just discovered prowler, so I'm not sure if this is the correct place to report this.
> 
> The version of prowler doesn't match the docker image tag:
> 
> ```
> $ docker run --rm toniblyx/prowler:2.9.0 -V
> Prowler 2.8.1-25March2022
> ```
> 
> I'm happy to open a bug report if needed, thanks!

Thank you for notifying this @dsech !! It's fixed now.
